lib/qesapdeployment.pm

```diff
@@ -91,7 +103,115 @@ sub qesap_get_deployment_code {
         push(@log_files, QESAP_GIT_CLONE_LOG);
         assert_script_run("set -o pipefail ; $git_clone_cmd | tee " . QESAP_GIT_CLONE_LOG, quiet => 1);
     }
+    assert_script_run("ln -s " . TERRAFORM_DIR . "/aws " . TERRAFORM_DIR . "/ec2");
+}
+
+
+=head3 qesap_upload_logs
+
+    Collect and upload logs present in @log_files.
+=cut
+
+sub qesap_upload_logs {
+    record_info("Uploading logfiles", join("\n", @log_files));
+    for my $file (@log_files) {
+        upload_logs($file, failok => 1);
+    }
+    # Remove uploaded files from arrays
+    @log_files = ();
+}
+
+
+=head3 qesap_yaml_replace
+
+    Replaces yaml config file variables with parameters defined by OpenQA testode, yaml template or yaml schedule.
+    Openqa variables need to be added as a hash with key/value pair inside %run_args{openqa_variables}.
+    Example:
+        my %variables;
+        $variables{HANA_SAR} = get_required_var("HANA_SAR");
+        $variables{HANA_CLIENT_SAR} = get_required_var("HANA_CLIENT_SAR");
+        $run_args{openqa_variables} = %variables;
```

Comment:
is line 133 of this example needed?

Reply:
Oh thanks. Good catch... that was a remainder of the old code.
Removed.